### DISCUSSION TITLE
Is it possible to set a twitter:image and og:image for search page?

### DISCUSSION THREAD

**Discussion by Natenom** [razonyang/hugo-theme-bootstrap#744] [Q&A] [answered]
Currently the search page twitter:image and og:image is the default logo.

I would like it to be a special image, for example a magnifying glass or something like that.

Reason: Linking in social media should show that this is a search result.

**Comment by razonyang** [comment, marked as answer]
Hi, those are pages, you can do it as same as your other pages, either specify the `images` in front matter(`content/search/index.md`) or save the image into page folder.

```toml
images = [
    "/images/search-icon.png"
]
```

> You need to save your images into `/static/images`.

Or

```bash
$ tree content/search                
content/search
├── _index.fr.md
├── _index.ja.md
├── _index.md
├── _index.zh-hans.md
├── _index.zh-hant.md
└── feature.png
```

**Comment by Natenom** [comment]
Thank you.

Tried it and it works :)

But the path is always related to / instead of being relative.

So
```
images:
  - search.webp
```
results in `https://blog.natenom.com/search.webp`

So I set it to 
```
images:
  - /images/search.webp
```

And moved the file to `/static/images/`.

**Reply by razonyang** [reply]
Corrected the code snippet.
